app/models/compiled_result.rb

```diff
@@ -0,0 +1,20 @@
+class CompiledResult
```

Comment:
Is `CompiledResult` meant to get stored anywhere? I don't see it in the db after creating a test.

Reply:
Never mind. I was looking in the wrong db, as it gets renamed to `cypress_v5_development` in this PR.